pkg/apis/autoscaling/v1alpha1/pa_lifecycle.go

```diff
@@ -215,6 +215,12 @@ func (pas *PodAutoscalerStatus) MarkScaleTargetInitialized() {
 	podCondSet.Manage(pas).MarkTrue(PodAutoscalerConditionScaleTargetInitialized)
 }
 
+// MarkScaleTargetNotInitialized marks the PA's PodAutoscalerConditionScaleTargetInitialized
+// condition true.
+func (pas *PodAutoscalerStatus) MarkScaleTargetNotInitialized(reason, message string) {
+	podCondSet.Manage(pas).MarkFalse(PodAutoscalerConditionScaleTargetInitialized, reason, message)
+}
+
```

Comment:
We should double check usages of this condition. Because before it would always be `Unknown=>(True|False)` and then remain unchanged.

I can't recall if there's code that assumes that it never changes.

Reply:
Maybe we might want to introduce a new condition - to maybe surface subsequent scaling issues

Reply:
> I can't recall if there's code that assumes that it never changes.

Tests are not covering revision transitions? I checked the pa status propagation for the revision reconciliation and we have specific cases where this matters. I can take a look again if there is a scenario where this might be a problem. In general we should be able to set this to False (for whatever reason) and then any reconciliation should take into consideration that condition.